seeing this build error on tuo (`oc+gssapi`)

```
     5     ==> oc: Executing phase: 'build'
     6     ==> [2026-04-08-12:06:34.800490] /usr/WS2/melone1/spack/opt/spack/linux-zen4/go-1.26.1-pftu6kn3pipvwbpkrsvj3jc7na4ehyxl/bin/go build -p 100 -modcacherw -ldflags '-s -w' -o oc
           -tags 'include_gcs include_oss containers_image_openpgp gssapi'
     7     # github.com/openshift/oc/pkg/helpers/authchallengers
  >> 8     ../../pkg/helpers/authchallengers/negotiator_gssapi.go:24:14: undefined: gssapi.Lib
  >> 9     ../../pkg/helpers/authchallengers/negotiator_gssapi.go:31:15: undefined: gssapi.Name
  >> 10    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:33:14: undefined: gssapi.CtxId
  >> 11    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:42:15: undefined: gssapi.CredId
  >> 12    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:81:96: undefined: gssapi.GSS_C_INITIATE
  >> 13    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:118:28: undefined: gssapi.Buffer
  >> 14    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:127:14: undefined: gssapi.ErrContinueNeeded
  >> 15    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:158:47: undefined: gssapi.Lib
  >> 16    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:176:29: undefined: gssapi.Load
  >> 17    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:176:42: undefined: gssapi.Options
  >> 18    ../../pkg/helpers/authchallengers/negotiator_gssapi.go:176:42: too many errors
```

--- a/repos/spack_repo/builtin/packages/oc/package.py
+++ b/repos/spack_repo/builtin/packages/oc/package.py
@@ -18,16 +18,11 @@ class Oc(GoPackage):
     license("Apache-2.0")
 
     version("main", branch="main")
-    version(
-        "4.21.0-202601121715",
-        tag="openshift-clients-4.21.0-202601121715",
-        commit="345800dc3c4164fbca313c1cbfb383f262547903",
-    )
+    version("4.21.0-202601121715", commit="345800dc3c4164fbca313c1cbfb383f262547903")
 
     variant("gssapi", default=False, description="Enable GSSAPI authentication support")
 
-    conflicts("+gssapi", when="platform=darwin os=tahoe")
-
+    depends_on("c", type="build")
     depends_on("go@1.22.5:", type="build", when="@4.20:")
 
     depends_on("gnupg")
@@ -36,7 +31,7 @@ class Oc(GoPackage):
     depends_on("krb5", when="+gssapi")
 
     build_directory = "cmd/oc"
-    cgo_enable = True
+    cgo_enabled = True
 
     @property
     def build_args(self):

